pytorch_lightning/tuner/tuning.py

```diff
@@ -43,6 +44,12 @@ def _tune(
 
         self.trainer.strategy.connect(model)
 
+        is_tuning = self.trainer.auto_scale_batch_size or self.trainer.auto_lr_find
+        if self.trainer._accelerator_connector.is_distributed and is_tuning:
+            raise MisconfigurationException(
+                f"`trainer.tune()` is not supported with `Trainer(strategy={self.trainer.strategy.strategy_name!r})`."
+            )
```

Comment:
It sounds like it will never be supported in the future, but there's some possibility that it will be, right?

```suggestion
            raise MisconfigurationException(
                f"`trainer.tune()` is currently not supported with `Trainer(strategy={self.trainer.strategy.strategy_name!r})`."
            )
```